gst/mqtt/mqttsink.c

```diff
@@ -929,10 +929,10 @@ gst_mqtt_sink_set_caps (GstBaseSink * basesink, GstCaps * caps)
   ret = gst_caps_replace (&self->in_caps, caps);
 
   if (ret && gst_caps_is_fixed (self->in_caps)) {
-    char *caps_str = gst_caps_to_string (caps);
+    gchar *caps_str = gst_caps_to_string (caps);
```

Comment:
```suggestion
    g_autofree gchar *caps_str = gst_caps_to_string (caps);
    
    self->self->mqtt_msg_hdr.gst_caps_str = g_strdup (caps);
```

https://docs.gtk.org/glib/func.strlcpy.html

Reply:
self->mqtt_msg_hdr.gst_caps_str is fixed size string array, cannot set allocated memory.

https://github.com/nnstreamer/nnstreamer/blob/eb54486d4d17f1146fea7cfb6d304d65ebd8dbd8/gst/mqtt/mqttcommon.h#L59

Reply:
@gichan-jang @jaeyun-jung Got it!